packages/analytics/analytics-metric-provider/src/composables/useMetricFetcher.ts

```diff
@@ -173,7 +173,7 @@ export default function useMetricFetcher(opts: MetricFetcherOptions): FetcherRes
   })
 
   return {
-    isLoading: computed(() => STATE.PENDING === metricRequestState.value),
+    isLoading: computed(() => [STATE.PENDING, STATE.VALIDATING, STATE.VALIDATING_HAS_DATA].includes(metricRequestState.value)),
```

Comment:
Hmmm.  This makes the auto-refresh UX very noisy -- the page drops to loading states every time the charts update.  I'm not aware of many uses of auto-refresh right now, so this is probably an acceptable trade-off right now, but I'll poke Christian and Helen.

Reply:
You're right. I removed this change, I don't think refresh should trigger the loading state unless Christian/Helen really want it, so we can add that back in if they want that.